topology/topology.go

```diff
@@ -82,38 +69,24 @@ type Decrypter interface {
 }
 
 type topologyProvider struct {
-	client       minio.Client
-	documentName string
-	bucketName   string
-	decrypter    Decrypter
+	url       string
+	decrypter Decrypter
 }
 
 func (t *topologyProvider) NetworkTopology() (NetworkTopology, error) {
-	ctx := context.Background()
-
-	obj, err := t.client.GetObject(ctx, t.bucketName, t.documentName, minio.GetObjectOptions{})
+	resp, err := http.Get(t.url)
```

Comment:
What is that URL supposed to be? Should we keep our IPFS node or fetch from some providers like Cloudflare for example?

Reply:
URL is basically an URL to that file on some gateway (ie. ipfs.io)